lib/puppet/indirector/indirection.rb

```diff
@@ -285,7 +285,7 @@ def save(instance, key = nil, options={})
     result = terminus.save(request)
 
     # If caching is enabled, save our document there
-    cache.save(request) if cache?
+    cache.save(request) if cache? && !request.ignore_cache_save?
```

Comment:
This line and line 164 are not covered by spec tests in spec/unit/indirector/indirection_spec.rb.  Is this OK?